updater/spec/dependabot/updater/operations/group_update_all_versions_spec.rb

```diff
@@ -81,6 +81,40 @@
     end
   end
 
+  context "when there are two overlapping groups" do
+    let(:job_definition) do
+      job_definition_fixture("bundler/version_updates/group_update_all_overlapping_groups")
+    end
+
+    let(:dependency_files) do
+      original_bundler_files
+    end
+
+    before do
+      stub_rubygems_calls
```

Comment:
Does this need to be cleared/reset after the tests?

Or does it already register a cleanup callback internally?

Reply:
Good spot, it is ultimately just `webmock` stubs which are plugged into RSpec to clear afterwards automagically.